tests/rule_based_profiler/test_profiler_run_anonymizer.py

```diff
@@ -0,0 +1,328 @@
+from typing import Dict
+
+import pytest
+
+from great_expectations.core.batch import BatchRequest
+from great_expectations.core.usage_statistics.anonymizers.profiler_run_anonymizer import (
+    ProfilerRunAnonymizer,
+)
+from great_expectations.rule_based_profiler.rule_based_profiler import RuleBasedProfiler
+
+
+@pytest.fixture
+def profiler_run_anonymizer() -> ProfilerRunAnonymizer:
+    # Standardize the salt so our tests are deterinistic
+    salt: str = "00000000-0000-0000-0000-00000000a004"
+    anonymizer: ProfilerRunAnonymizer = ProfilerRunAnonymizer(salt=salt)
+    return anonymizer
+
+
+@pytest.fixture
+def usage_stats_profiler_config() -> dict:
```

Comment:
These fixtures are great - can you add a version of the single and multiple rule config that also has custom class names in addition to GE supplied classes for DomainBuilder, ParameterBuilder, ExpectationConfigurationBuilder and custom Expectations - and test that the custom names are anonymized and the GE supplied are not?